src/builtins/shared.rs

```diff
@@ -495,6 +503,7 @@ pub fn builtin_get_desc(name: &wstr) -> Option<&'static wstr> {
         _ if name == "exit" => wgettext!("Exit the shell"),
         _ if name == "false" => wgettext!("Return an unsuccessful result"),
         _ if name == "fg" => wgettext!("Send job to foreground"),
+        _ if name == "fish_key_reader" => wgettext!("explore what characters keyboard keys send"),
```

Comment:
why is fish_key_reader added here but fish_indent isn't?

Reply:
Simple: I forgot.